trestle/core/commands/merge.py

```diff
@@ -14,119 +14,131 @@
+        """Merge operations.
+
+        It returns a plan for the operation
+        """
+        element_path_list = element_path.get_full_path_parts()
+        target_model_alias = element_path_list[-1]
+        """1. Load desination model into a stripped model"""
+        # Load destination model
+        destination_model_alias = element_path_list[-2]
+        # Destination model filename
+        destination_model_filename = Path(f'{utils.classname_to_alias(destination_model_alias, "json")}.json')
+        destination_model_type, _ = fs.get_stripped_contextual_model(destination_model_filename.absolute())
+
+        destination_model_object = destination_model_type.oscal_read(destination_model_filename)
+        """1.5. If target is wildcard, load distributed destrination model and replace destination model."""
+        # Handle WILDCARD '*' match. Return plan to load the destination model, with it's distributed attributes
+        if target_model_alias == '*':
+            merged_model_type, merged_model_alias, merged_model_instance = load_distributed.load_distributed(
+                destination_model_filename)
+            plan = Plan()
+            reset_destination_action = CreatePathAction(destination_model_filename.absolute(), clear_content=True)
+            write_destination_action = WriteFileAction(
+                destination_model_filename, Element(merged_model_instance), content_type=FileContentType.JSON
+            )
+            delete_target_action = RemovePathAction(Path(merged_model_alias).absolute())
+            plan: Plan = Plan()
+            plan.add_action(reset_destination_action)
+            plan.add_action(write_destination_action)
+            plan.add_action(delete_target_action)
+            return plan
+
+        # Get destination model without the target field stripped
+        merged_model_type, merged_model_alias = fs.get_stripped_contextual_model(
+            destination_model_filename.absolute(),
+            aliases_not_to_be_stripped=[target_model_alias])
+        """2. Load Target model. Target model could be stripped"""
+        try:
+            target_model_type = utils.get_target_model(element_path_list, merged_model_type)
+        except Exception as e:
+            raise err.TrestleError(
+                f'Target model not found. Possibly merge of the elements not allowed at this point. {str(e)}'
+            )
+        # target_model filename - depends whether destination model is decomposed or not
+        if (Path(os.getcwd()) / destination_model_alias).exists():
+            target_model_path = f'{os.getcwd()}/{destination_model_alias}/{target_model_alias}'
+        else:
+            target_model_path = target_model_alias
+
+        # if target model is a file then handle file. If file doesn't exist, handle the directory,
+        # but in this case it's a list or a dict collection type
+        if (Path(f'{target_model_path}.json')).exists():
```

Comment:
You  have made many assumptions of json. at least parameterize this so it's easy to fix later.

Reply:
There's an issue for it - I'll handle it with the issue